### PR TITLE
nandika-shendre.md

### PR DESCRIPTION
**Pull request by nandika-shendre**
[Rawgit link]
(https://github.com/nandika-shendre/nandika-shendre.github.io/blob/nandika-shendre-OLE/pages/vi/profiles/nandika-shendre.md)

### CODE REVIEW

**Comment by kylemathias** [comment]
Your page looks fine. When you link a rawgit.com link, the url should show the actual representation. here is the example for your profile page. https://rawgit.com/nandika-shendre/nandika-shendre.github.io/nandika-shendre-OLE/#!pages/vi/profiles/nandika-shendre.md

**Comment by xyb994** [comment]
Looks good overall but the LinkedIn url is not clickable in [MDwiki's rendering](https://rawgit.com/nandika-shendre/nandika-shendre.github.io/nandika-shendre-OLE/#!pages/vi/profiles/nandika-shendre.md).
You might want to use `[]()` or `<>` with the url.

**Comment by kylemathias** [comment]
Yes, try to make the LinkedIn URL a markdown element. After that then we can merge.

File: pages/vi/profiles/nandika-shendre.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Nandika Shendre
+## Los Angeles, CA / PST Windows 10
+### I graduate in Dec 2017 from **University of Cincinnati.**
+### Main area of interest is DevOps.
+### I have experience in Python and Node JS.
+
+
+### LinkedIn URL: <www.linkedin.com/in/nandikashendre>
```
Comment (thesiege208) [inline pages/vi/profiles/nandika-shendre.md]: Your LinkedIn link isn't visible. Github doesn't automatically hyperlink `<links in angle brackets>`. You should use `[this](format)` instead.

**Comment by lorugant** [comment]
@nandika-shendre Are you still working on this?

**Pull request closed without merging**